Add dummy dialogs for town citizens.

Predelnik opened the pull request:
The following basic things yet need to be done:
- ~~Dialogs currently do not block clicks on the rest of the screen.~~
- ~~Mouse interaction~~
- ~~Escape button for now does not work because it should not call main menu in case of dialog (which is sort of similar to first item)~~
- Long press on arrow button should trigger fast scrolling between items.

But more or less it should fix #273

Predelnik commented:
So since PR is not yet merged I added fixes for everything I mentioned in the comment except long press. To support long press perhaps possibly I would have to refrain from using nuklear input but I don't think it's a significant feature anyway.

wheybags commented on apps/freeablo/fagui/dialogmanager.cpp:
```diff
@@ -0,0 +1,271 @@
+    void DialogManager::talkOgden()
+    {
+        DialogData d;
+        d.header({"Welcome to the", "Rising Sun"});
```
I'd prefer not to have this text in here. If we're to have placeholder text, just some lorem ipsum would be better

Predelnik replied:
Sorry, I don't quite get that. You mean I should extract strings from exe? But before that it was in writtine in .rml all the same, what's the difference?

wheybags replied:
Yeah, I didn't know it was written in the rml before :s
IMO it's better to have placeholder text so we know it's not implemented, than to have the real text and forget it's hardcoded instead of loaded properly.

Predelnik commented:
I've done it.
To not create a ton of variables I just added a dictionary of strings for each character with (I hope) understandable keys. Variable for npcs in ini that are prefixed with `talk#` are processed automatically and added to their corresponding dictionaries.

wheybags commented:
Nice, thanks!
